script/generate-mime.rb

```diff
@@ -139,24 +140,24 @@ def get_matches(parent)
 puts "  # :nodoc:"
 puts "  EXTENSIONS = {"
 extensions.keys.sort.each do |key|
-  puts "    '#{key}' => '#{extensions[key]}',"
+  puts "    '#{key}' => '#{extensions[key]}'.freeze,"
```

Comment:
why not just put the pragma in the beginning?

Reply:
I'm not sure what you mean :(